docs/android/features/_shared/_custom-views/recipeDetail/RecipeDetailCustomization.mdx

```diff
@@ -0,0 +1,134 @@
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Header from './RecipeDetailHeader.mdx';
+import Info from './RecipeDetailInfo.mdx';
+import SponsorBanner from "./RecipeDetailSponsorBanner.mdx";
+import Tags from "./RecipeDetailTags.mdx"
+import Swapper from "./RecipeDetailSwapper.mdx"
+import Footer from "./RecipeDetailFooter.mdx"
+import ProductListHeader from "./RecipeDetailProductListHeader.mdx"
+import Product from "./RecipeDetailProduct.mdx"
+import OftenDeleted from "./RecipeDetailOftenDeleted.mdx"
+import Unavailable from "./RecipeDetailUnavailable.mdx"
+import Ingredients from "./RecipeDetailIngredients.mdx"
+import Steps from "./RecipeDetailSteps.mdx"
+
+## RecipeDetail
+
+RecipeDetails is one of the most customizable component, It has two customizations state **success** & **loading:**
+
+### Loading 
+
+Loading is of type [**LoaderConfig**](../../base-component/customize-views#loading) you can eather create a specific 
+template for RecipeDetails otherwise the default template define in **defaultViews** would be use.
+
+
+`LoaderParameters` is an empty object for now, but tomorrow we'll be able to add properties to it, in order to avoid breaking changes in future versions. 
```

Comment:
```suggestion
`LoaderParameters` is an empty object for now, but we have the option to add properties to it in the future in order to avoid breaking changes. 
```